libcst/_typed_visitor.py

```diff
@@ -4247,6 +4229,41 @@ def visit_Subscript_whitespace_after_value(self, node: "Subscript") -> None:
     def leave_Subscript_whitespace_after_value(self, node: "Subscript") -> None:
         pass
 
+    def visit_SubscriptElement(self, node: "SubscriptElement") -> Optional[bool]:
+        return self.visit_ExtSlice(node)
+
+    def visit_SubscriptElement_slice(self, node: "SubscriptElement") -> None:
+        self.visit_ExtSlice_slice(node)
+
+    def leave_SubscriptElement_slice(self, node: "SubscriptElement") -> None:
+        self.leave_ExtSlice_slice(node)
+
+    def visit_SubscriptElement_comma(self, node: "SubscriptElement") -> None:
+        self.visit_ExtSlice_comma(node)
+
+    def leave_SubscriptElement_comma(self, node: "SubscriptElement") -> None:
+        self.leave_ExtSlice_comma(node)
+
+    @mark_no_op
+    def visit_ExtSlice(self, node: "ExtSlice") -> Optional[bool]:
+        pass
+
+    @mark_no_op
+    def visit_ExtSlice_slice(self, node: "ExtSlice") -> None:
+        pass
+
+    @mark_no_op
+    def leave_ExtSlice_slice(self, node: "ExtSlice") -> None:
+        pass
+
+    @mark_no_op
+    def visit_ExtSlice_comma(self, node: "ExtSlice") -> None:
+        pass
+
+    @mark_no_op
+    def leave_ExtSlice_comma(self, node: "ExtSlice") -> None:
+        pass
```

Comment:
Can we call `warnings.warn(..., DeprecationWarning)` in those deprecated visit and leave function to provide warning to users?

Reply:
After rethink about it, for showing the warning, we'll need some work on at_visit/at_leave which doesn't seem worth it.